chanbackup/single.go

```diff
@@ -36,6 +36,10 @@ const (
 	// implicitly denotes that this channel uses the new anchor commitment
 	// format.
 	AnchorsCommitVersion = 2
+
+	// ZeroFeeSecondLevelCommitVersion is a version that denotes this
+	// channel isusing the zero-fee second-level anchor commitment format.
+	ZeroFeeSecondLevelCommitVersion = 3
```

Comment:
We don't strictly need a new version, since on recovery we don't handle HTLCs anyway.